cmake/ConfigFileSetting.cmake

```diff
@@ -263,6 +264,17 @@ nrn_configure_file(nmodlconf.h .)
 nrn_configure_file(nrnunits.lib share/lib OUTPUT share/nrn/lib)
 nrn_configure_file(nrn.defaults share/lib OUTPUT share/nrn/lib)
 nrn_configure_file(constants.py share/lib/python/neuron/rxd)
+if (MINGW)
+  set(dos_marshal_dir "c:\\marshalnrn")
+  nrn_configure_file(nrnsetupmingw.nsi src/mswin)
+  nrn_configure_file(AddToPath.nsh src/mswin)
+  nrn_configure_file(EnvVarUpdate.nsh src/mswin)
+  set(CC x86_64-w64-mingw32-gcc.exe)
```

Comment:
why this is hard-coded? a comment would be useful.

Reply:
Has to be just the program name and not a path as it will be used  after the setup.exe install in a place chosen by the user.  Added  a comment.

Reply:
but the compiler name is always `x86_64-w64-mingw32-gcc.exe`? (note that [get_filename_component](https://cmake.org/cmake/help/v3.0/command/get_filename_component.html) is used to retried the filename from full path)